staging/src/k8s.io/component-base/featuregate/testing/feature_gate.go

```diff
@@ -18,18 +18,34 @@ package testing
 
 import (
 	"fmt"
-	"testing"
```

Comment:
why redeclare the testingTB interface locally? I'd like to keep the `testing` dep so this package cannot get imported into non-test code

Reply:
Done